addon/components/code-inline.js

```diff
@@ -1,3 +1,5 @@
+/* global Prism */
```

Comment:
Yay! I was helping out with another addon, and didn't like the way they showed the code. So I whipped this together.